tests/test_bookings.py

```diff
@@ -0,0 +1,220 @@
+import pytest
+from fastapi.testclient import TestClient
+from main import app
+from unittest.mock import patch
+import datetime
+from utils import users as users_utils
+from schemas import bookings as bookings_schema
+
+
+booking_data = {
+    'room_number': 1,
+    'guest_id': 1,
+    'check_in': str(datetime.datetime.now() - datetime.timedelta(hours=1)),
+    'check_out': str(datetime.datetime.now() + datetime.timedelta(days=10)),
+    'description': "descr of booking",
+}
+booking_data_add = {
+    "id": 11,
+    "is_paid": False,
+    "is_active": True,
+    "client_review": "review",
+    "updated_at": datetime.datetime.now(),
+    "created_at": datetime.datetime.now(),
+}
+guest_data = {
+    'name': 'name_of_guest',
+    'email': 'name@guest.com',
+    'phone': '98746543210',
+}
+roomtype_data = {
+    'type_name': 'super-puper',
+    'price': 999.9,
+    'description': 'Best room',
+}
+
+
+@pytest.fixture
+def client(client: TestClient):
+    '''Update global variable'''
+
+    # skips the authentication
+    def skip_auth():
+        pass
+    app.dependency_overrides[users_utils.get_current_user] = skip_auth
+
+    yield client
+
+
+def test_routers_bookings(client):
```

Comment:
what exactly are you testing here?
there are different endpoints in this test, and if it fails it's hard to understand which one is broken. Also you test only positive scenarios here.

1 unit test usually covers 1 function and for better readability its name should be test_function_name_some_conditions: test_create_booking_insufficient_permissions etc